### PR TITLE
[TiUP] add support check the yaml file before scale out

### PR DESCRIPTION
**Pull request by shichun-0415**
### What is changed, added or deleted? (Required)
[TiUP v1.9.0](https://github.com/pingcap/tiup/releases/tag/v1.9.0) supports checking the yaml file before scale out. This feature is very useful and should be documented.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/12273
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by srstack — APPROVED** [review]

File: tiup/tiup-component-cluster-check.md [hunk 1]
```diff
@@ -143,17 +145,34 @@ tiup cluster check <topology.yml | cluster-name> [flags]
...
-若选择的格式为 `tiup cluster check <cluster-name>` 则必须加上该选项：`tiup cluster check <cluster-name> --cluster`。
+    ```shell
+    tiup cluster check <topology.yml | cluster-name> [flags]
+    ```
 
-该选项的数据类型为 `BOOLEAN`。该选项默认关闭，默认值为 `false`。在命令中添加该选项，并传入 `true` 值或不传值，均可开启此功能。
+> **注意：**
+>
+> - 若选择的格式为 `tiup cluster check <cluster-name>`，则必须加上该选项：`tiup cluster check <cluster-name> --cluster`。
+> - tiup-cluster 也支持检查已部署集群的扩容拓扑文件，命令格式：
```
Comment (qiancai) [inline tiup/tiup-component-cluster-check.md:171]:
```suggestion
> - `tiup cluster check` 也支持检查已部署集群的扩容拓扑文件，命令格式：
```

File: tiup/tiup-component-cluster-check.md [hunk 2]
```diff
@@ -143,17 +145,34 @@ tiup cluster check <topology.yml | cluster-name> [flags]
...
+> tiup cluster check <cluster-name> scale-out.yaml --cluster --apply --user root [-p] [-i /home/root/.ssh/gcp_rsa]
+>```
 
-tiup-cluster 支持对未部署的集群进行检查，也支持对已部署的集群进行检查，命令格式：
+### --cluster
 
-```shell
-tiup cluster check <topology.yml | cluster-name> [flags]
-```
+- 对部署的集群进行检查。
```
[the remark this answers is not in the document]
Comment (shichun-0415) [reply tiup/tiup-component-cluster-check.md:158]: 已部署和未部署的集群，都支持检查。或许需要把这一行的”对部署的集群进行检查“改为”对已部署和未部署的集群进行检查“？

File: tiup/tiup-component-cluster-check.md [hunk 3]
```diff
@@ -143,17 +145,34 @@ tiup cluster check <topology.yml | cluster-name> [flags]
...
+- 数据类型：`BOOLEAN`
+- 默认值：`false`
+- 在命令中添加该选项，并传入 `true` 值或不传值，均可开启此功能。
+- 命令格式（已部署和需要扩容的集群都适用）：
```
Comment (shichun-0415) [inline tiup/tiup-component-cluster-check.md:162]:
```suggestion
- 命令格式：
```

File: tiup/tiup-component-cluster-check.md [hunk 4]
```diff
@@ -143,17 +145,34 @@ tiup cluster check <topology.yml | cluster-name> [flags]
...
+> tiup cluster check <cluster-name> scale-out.yaml --cluster --apply --user root [-p] [-i /home/root/.ssh/gcp_rsa]
+>```
 
-tiup-cluster 支持对未部署的集群进行检查，也支持对已部署的集群进行检查，命令格式：
+### --cluster
 
-```shell
-tiup cluster check <topology.yml | cluster-name> [flags]
-```
+- 对部署的集群进行检查。
```
Comment (shichun-0415) [inline tiup/tiup-component-cluster-check.md:158]:
```suggestion
- 对已部署的集群进行检查。
```

File: tiup/tiup-component-cluster-check.md (hunk 3 above)
Comment (shichun-0415) [reply tiup/tiup-component-cluster-check.md:162]: 需要扩容的集群也算“已部署的集群”，所以似乎可以去掉这里。Note 里已经交代了支持检查需要扩容文件，不会引起误解了。

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**